river/Seat.zig

```diff
@@ -64,6 +64,9 @@ prev_mode_id: usize = 0,
 /// is currently available for focus.
 focused_output: *Output,
 
+/// Has the output focus changed since the last transaction?
+output_focus_changed: bool = false,
```

Comment:
Should `Seat` perhaps also have a `State` struct with a `pending` and `current` instance?